### PR TITLE
VRTCAL Bid Adapter : updated user sync support

### PR DESCRIPTION
**Pull request by vrtcal-dev**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

### CODE REVIEW

File: modules/vrtcalBidAdapter.js [hunk 1]
```diff
@@ -148,7 +150,34 @@ export const spec = {
...
+  },
+
+  getUserSyncs: function(syncOptions, serverResponses, gdprConsent = {}, uspConsent = '', gppConsent = {}) {
+    const syncs = [];
+    const gdprFlag = `&gdpr=${gdprConsent.gdprApplies ? 1 : 0}`;
+    const gdprString = `&gdpr_consent=${encodeURIComponent((gdprConsent.consentString || ''))}`;
+    const usPrivacy = `&us_privacy=${encodeURIComponent(uspConsent)}`;
+    const gpp = gppConsent.gppString ? gppConsent.gppString : '';
+    const gppSid = Array.isArray(gppConsent.applicableSections) ? gppConsent.applicableSections.join(',') : '';
+    var vrtcalSyncURL = ''
```
Comment (jsnellbaker) [inline modules/vrtcalBidAdapter.js:162]: For consistency, can you change this `var` to a `let`?

Comment (vrtcal-dev) [reply modules/vrtcalBidAdapter.js]: Adjustment made

**Comment by jsnellbaker** [comment]
Hi @vrtcal-dev 

Is there a way to test your bidder with a locally hosted test page?  I was trying to test the updates, however I encountered a CORS error with the endpoint due to the locally hosted hello_world.html I modified.

I saw the iframe userSync URL try to execute, but it was canceled by the browser.  

Screen Shot 2023-10-11 at 1 13 39 PM

**Comment by vrtcal-dev** [comment]
> Hi @vrtcal-dev
> 
> Is there a way to test your bidder with a locally hosted test page? I was trying to test the updates, however I encountered a CORS error with the endpoint due to the locally hosted hello_world.html I modified.
> 
> I saw the iframe userSync URL try to execute, but it was canceled by the browser.
> 

Thank you for your screenshot as it was helpful in us determining why you were getting CORS errors for our bidder at all. You were seeing this due to the server back end not setting the CORS * allowance for non-bid returns. This has been corrected, so you should be able to retest without any CORS errors.

**Comment by jsnellbaker** [comment]
As an update - I'm not seeing the CORS error on the endpoint.  But I'm still seeing the iframe user sync URL getting canceled per the Network tab.  I'm not sure if it's related to that there was no bid response.  

See below for a reference:
Screen Shot 2023-10-12 at 8 12 34 AM

**Review by jsnellbaker — APPROVED** [review]
If you happened to see my deleted comment, please ignore - there was an issue with my Chrome browser at the time.

LGTM

**Pull request merged**